### PR TITLE
HW 01 Shell student 28

### PR DESCRIPTION
**Pull request by acanna**

### CODE REVIEW

**Comment by acanna** [comment]
HW 1 Student 28

File: 28/01/Makefile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+build:
+		gcc -c sh.c -o sh
+clean:
+		find ! -name sh.c ! -name Makeile -delete
```
Comment (krinkin) [inline 28/01/Makefile]: make clean deletes makefile (: do you know why?

File: 28/01/Makefile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+build:
+		gcc -c sh.c -o sh
```
Comment (krinkin) [inline 28/01/Makefile]: are you sure that sh will be executable?

**Review by krinkin — CHANGES_REQUESTED** [review]
требуются доработки

**Comment by acanna** [comment]
Fixed

**Pull request merged**